.circleci/config.yml

```diff
@@ -0,0 +1,167 @@
+defaults: &defaults
+  docker:
+    - image: buildpack-deps:xenial
+  working_directory: ~/
+
+test-defaults: &test-defaults
+  <<: *defaults
+  steps:
+    - checkout:
+        path: emscripten/
+    - attach_workspace:
+        # Must be absolute path or relative path from working_directory
+        at: ~/
+
+    - run: |
+        apt-get update
+        apt-get install -y python python-pip cmake build-essential openjdk-9-jre-headless
+        pip install --upgrade pip
+        pip install lit
+        EMCC_CORES=8 python emscripten/tests/runner.py $TEST_TARGET skip:ALL.test_sse1_full skip:ALL.test_sse2_full skip:ALL.test_sse3_full skip:ALL.test_ssse3_full skip:ALL.test_sse4_1_full skip:other.test_native_link_error_message skip:other.test_bad_triple
+
+version: 2
+jobs:
+  build:
+    <<: *defaults
+    steps:
+      - checkout:
+          path: emscripten/
+      - run: |
+          apt-get update
+          apt-get install -y python cmake
+          wget https://s3.amazonaws.com/mozilla-games/emscripten/releases/emsdk-portable.tar.gz
+          tar -xf emsdk-portable.tar.gz
+          pushd emsdk-portable
+          ./emsdk update
+          ./emsdk install latest
+          ./emsdk activate latest
+          popd
+          echo EMSCRIPTEN_ROOT="'~/emscripten/'" >> .emscripten
+          echo BINARYEN_ROOT="''" >> .emscripten
+          EMCC_CORES=8 python2 emscripten/embuilder.py build ALL
```

Comment:
do they have 8 physical cores? impressive :)

in the build output it looks like the test runner spawns 16 processes actually, not 8 (counting `(Emscripten: Running sanity checks)`) which is very strange. setting `EMCC_CORES` should set that (and it does for me locally)

Reply:
`(checking sanity from test runner)` occurs 8 times so EMCC_CORES is setting it. Hmm...

Reply:
>do they have 8 physical cores? impressive :)

The actual assigned cores are 2 but giving 4 or 8 slightly accelerates in my experience. Without the explicit `EMCC_CORES` emscripten thinks the machine has 32 cores, which causes memory issue.